src/main/kotlin/com/sexysisters/tojserverv2/domain/school/domain/School.kt

```diff
@@ -47,4 +51,10 @@ class School(
     fun birthdayValue() = birthday.value
     fun homePageAddressValue() = homePageAddress.value
     fun phoneValue() = phone.value
+    fun wallPaperValue() = wallpaper.value
+
+    fun updateWallpaper(wallpaper: String): String {
+        this.wallpaper = Wallpaper(wallpaper)
+        return wallPaperValue()
+    }
```

Comment:
포장클래스로 감싸는 로직이 도메인에 있으면 어색할 것 같습니다. 서비스에서 포장한 뒤 도메인에서는 바꿔주기만 하는게 좋을 것 같습니다!

```suggestion
    fun updateWallpaper(wallpaper: Wallpaper): String {
        this.wallpaper = wallpaper
        return wallPaperValue()
    }
```